traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/fragments/CatalogItemDetailsFragment.java

```diff
@@ -140,6 +140,32 @@ public void onClick(View v) {
             }
         });
 
+        wishListToggleImageButton = view.findViewById(R.id.imagebutton_wishlist_toggle);
+        wishListToggleImageButton.setSelected(false);
+        wishListToggleImageButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (wishListToggleImageButton.isSelected()) {
+                    Toast.makeText(getContext(), "Removing from wishlist is not implemented yet!", Toast.LENGTH_SHORT).show();
```

Comment:
This is not acceptable to be in a commit. Its broken and the commit should include both add and remove